setup.py

```diff
@@ -15,12 +15,12 @@
     version = version.split('/')[-1]
 
 # Template files for running on ATLAS.
-xaod_template_files = os.listdir('func_adl_xAOD/R21Code')
+xaod_template_files = os.listdir('func_adl_xAOD/template/atlas/r21')
```

Comment:
What about CMS templates?

Reply:
The setup.py doesn't reflect any of the cms code. I will do it in a future commit.

Reply:
@gordonwatts what is `xaod_template_files` variable used for?

Reply:
Ah - do not think it is needed any longer! Ops! See comment below!